.changeset/light-llamas-chew.md

```diff
@@ -0,0 +1,5 @@
+---
+"thirdweb": patch
+---
+
+Connect UI context code cleanup
```

Comment:
does this change any public api or fix a bug? if it's purely internal changes then we do not need a changeset for it

Reply:
No public API change - let me remove changeset